test/css/test-ruby.css

```diff
@@ -0,0 +1,110 @@
+@charset "UTF-8";
+/* # Module: Settings */
+/* ------------------ */
+/* Test: true-context [mixin] */
+/*   ✔ Confirm that there is currently no "fake" scope */
```

Comment:
I think these `✔` and `✖` chars were causing problems. I'll replace them with plain text.